dev/reference/tools/syncer.md

```diff
@@ -389,50 +389,22 @@ Before replicating data using Syncer, check the following items:
 
         If the result is `log_bin = OFF`, you need to enable the binlog. See the [document about enabling the binlog](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html).
 
-    2. Check whether the binlog format in MySQL is `ROW`.
-
-        Check the binlog format using the following command:
+    2. The binlog format must be `ROW` and the binary log must be written with `FULL` row images. Check both of these variables:
```

Comment:
There is some duplication, because the sentence above says:
>  If the result is `log_bin = OFF`, you need to enable the binlog.

Do you mind cleaning this up too to flow more naturally?

Reply:
Nevermind, discussed via chat. Looks good.